NeuralABC_tools/eigengame.py

```diff
@@ -39,13 +37,10 @@ def eigengame(data, n_components, epochs=100, learning_rate=0.1):
     
     Parameters:
     ----------------
-    data: a Numpy array containing the data to run PCA on
-    
-    n_components: the number of principal components to extract
-    
-    epochs: the number of iterations to calculate each eigenvector. Default = 100
-    
-    learning_rate: Learning rate of the algorithm. Default = 0.1
+    :param data: (array), required: a Numpy array containing the data to run PCA on
```

Comment:
can you specify the form of this array please? i.e., it is a 2d array with what expectations for each dimension?

Reply:
I made the description more specific